mail_tracking/models/mail_thread.py

```diff
@@ -35,38 +35,53 @@ def message_post(self, *args, **kwargs):
         """
         new_message = super().message_post(*args, **kwargs)
         email_cc = kwargs.get('cc')
+        toWrite = {}
         if email_cc:
-            new_message.sudo().write({
+            toWrite.update({
                 'email_cc': email_cc,
             })
+        # This is here because Odoo only store recipients that have a partner.
+        toWrite.update({
```

Comment:
What happens if no `to` key. Are we forcing a writing in the object any way? Please take the performance into account always as this is a very low level operation.

Reply:
Can exists a message without a 'to' defined?

Reply:
Yeah, indeed, but my problem is to perform an extra write each time. Isn't any `prepare` values function or pass this by context as intercept it in the write/create method for injecting it in the dictionary before dumping it to DB?